partner_phone_extension/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013-2014 Savoir-faire Linux
+#   (<http://www.savoirfairelinux.com>).
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
```

Comment:
Man! So much copyright holders for 1 line of code, isn't it?
🔥 this (see https://github.com/OCA/maintainer-tools/pull/260)